style/js/README.md

```diff
@@ -1,3 +1,190 @@
+JavaScript Style Guides
+=====
+
+Guides for programing JavaScript in style
+
+General
+---
+* Use [JSHint](http://jshint.com/)
+* Always use 'use strict'
+
+Parens and Braces
+---
+* Leave one space between 'if' and '('.
+
+```javascript
+if (something) {
+  //some work here;
+}
+```
+
+* Prefer this form for multiline if statments
+
+```javascript
+if (something) {
+  // do stuff
+  // even more stuff
+} else {
+  // some other stuff
+  // real stuff
+}
+```
+
+* On if statments, whenever it makes sense, you can omit the '{}'
+
+```javascript
+if (something)
+  return stuff;
+else
+  return otherStuff;
+```
+
+* Don't include spaces between 'function' and '('.
+
+```javascript
+function bar() {
+  return  foo;
+}
+
+var bar = function() {
+  return foo;
+}
+```
+
+Assignments and Declarations
+---
+* Initialize arrays with '[ ]'
+
+```javascript
+var array = [];
+```
+
+* Initialize objects with '{ }'
+
+```javascript
+var object = {};
+```
+
+* Leave one space after '{' and before '}' when initializing objects on the same line
+
+```javascript
+var map = { ready: 9, when: 4, "you are": 15 };
+```
+
+* Prefer this form for multiline object/array initialization
+
+```javascript
+var map = {
+  when: 4,
+  ready: 9,
+  "you are": 15
+};
+
+var map = [
```

Comment:
@zamith  What you are saying might be a good solution if the API Miguel is using accepts it that way. Probably won't. 
Regarding the rule, when I wrote it I was thinking in suggesting a good way, not enforcing anything . |f its not clear maybe we can reword it